components/wizard/overview.md

```diff
@@ -0,0 +1,153 @@
+---
+title: Overview
+page_title: Wizard Overview
+description: Overview of the Wizard for Blazor.
+slug: wizard-overview
+tags: telerik,blazor,wizard,overview
+published: True
+position: 0
+---
+
+# Wizard Overview
+
+The Wizard for Blazor component displays content in sequential, stepwise order. Each step of the Wizard has `Content` (`render fragment`), which can contain any type of HTML content including a [Form]({%slug form-overview%}) component.
+
+>caption The Wizard is separated into 3 main sections:
+* [Stepper]({%slug wizard-structure-stepper%})
+* [Content]({%slug wizard-structure-content%})
+* [Buttons]({%slug wizard-structure-buttons%})
+
+>caption In this article:
+
+* [Basics](#basics)
+* [Features](#features)
+
+## Basics
+
+#### To create a basic Telerik Wizard:
+
+1. Use the `TelerikWizard` tag
+1. under its `WizardSteps` tag, set and configure the desired `WizardStep` instances and include the desired content in their `Content` tag
+
+>caption Set up a basic Telerik Wizard. The result from the snippet.
```

Comment:
The result from the code snippet **below** or remove this sentence.

Reply:
Done